lib/appsignal/utils.rb

```diff
@@ -49,8 +118,7 @@ def encode_utf8(value)
           )
         end
       end
-
-      extend ClassMethods
```

Comment:
I think we can keep it in. The module construct does nothing currently, it's more clear this way.